From f74d1009ae615eadbeedbb3881f6231d457e5e09 Mon Sep 17 00:00:00 2001
From: Martin Pitt <martin.pitt@ubuntu.com>
Date: Wed, 22 Jun 2016 15:41:22 +0200
Subject: [PATCH] Drop obsolete initscripts dependency

This got introduced in commit 5daea839 for the /run migration.

Closes: #804956
---
 debian/changelog | 4 ++++
 debian/control   | 2 +-
 2 files changed, 5 insertions(+), 1 deletion(-)

diff --git a/debian/changelog b/debian/changelog
index 04e26b6..33e68f9 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,7 +1,11 @@
 console-log (1.2-1.0) UNRELEASED; urgency=low
 
+  [ Marc Haber ]
   * NOT YET RELEASED
 
+  [ Martin Pitt ]
+  * Drop obsolete initscripts dependency. (Closes: #804956)
+
  -- Marc Haber <mh+debian-packages@zugschlus.de>  Sun, 24 Nov 2013 16:12:09 +0100
 
 console-log (1.2-1) unstable; urgency=low
diff --git a/debian/control b/debian/control
index ca2601c..3dd80c6 100644
--- a/debian/control
+++ b/debian/control
@@ -10,7 +10,7 @@ Vcs-Browser: http://git.debian.org/?p=collab-maint/console-log.git;a=summary
 
 Package: console-log
 Architecture: all
-Depends: less, daemon, kbd, passwd (>= 1:4.0.3-23), adduser (>> 3.51), initscripts (>= 2.88dsf-13.3), ${misc:Depends}
+Depends: less, daemon, kbd, passwd (>= 1:4.0.3-23), adduser (>> 3.51), ${misc:Depends}
 Description: Puts logfile pagers on virtual consoles
  console-log keeps logfile pagers open on virtual consoles. The
  default configuration puts exim's mainlog on tty8, and the syslog on
-- 
2.8.1

